tslint.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Hmm, why do we need a lint config for TypeScript?

Reply:
Actually, I didn't use it. I'll delete it